packages/react-components/react-tabster/src/focus/createCustomFocusIndicatorStyle.ts

```diff
@@ -31,10 +31,10 @@ export function createCustomFocusIndicatorStyle<TStyle extends GriffelStyle | Gr
 ): TStyle extends GriffelStyle ? GriffelStyle : GriffelResetStyle {
   return {
     ':focus': {
-      outlineStyle: enableOutline ? undefined : 'none',
+      outlineColor: enableOutline ? undefined : 'transparent',
```

Comment:
So I just realized this utility is in our public API 😱. 

So I had a thought about both of our utilities: `createFocusOutline` and `createCustomFocusIndicatorStyle`.
We should disable `outline` in the `createFocusOutline` utility only, since the pseudo element based outline should not use the native outline (we don't want 2 focus rings). Whereas I don't see any reason to actually override `outline` at all in the custom focus indicator, and we should let users of `createCustomFocusInidicatorStyle` to do the necessary changes to `outline`

This would mean that `enableOutline` will no longer used, so we can deprecate the option without removing it to avoid breaking the API

Reply:
That makes sense to me -- If someone really needs `outline` styles for some reason (e.g. they're using it for something non-focus-related?), they always have the out of using `createCustomFocusInidicatorStyle`.